bin/radical-pilot-agent-funcs

```diff
@@ -105,14 +113,15 @@ class Executor(object):
 
         # start one worker per core
         if not self._nw:
-            self._nw = mp.cpu_count() 
+            self._nw = mp.cpu_count()
 
         self._log.debug('#workers: %d', self._nw)
 
         self._workers = list()
         for i in range(self._nw):
             wid  = '%s.%03d' % (self._uid, i)
             proc = mp.Process(target=self._work, args=[self._uid, wid])
+            proc.daemon = True
```

Comment:
Can you explain why that was added?  I think it explains the lingering processes we saw on the Jetstream VM.

Reply:
This was added by you under the branch `fix_funcs`.

Reply:
Ah :-D  Let me look through that code again...

Reply:
I think this is fixed by now?

Reply:
yes